v3/plugins/ms-vscode/vscode-git/latest/meta.yaml

```diff
@@ -0,0 +1,15 @@
+publisher: ms-vscode
+apiVersion: v2
+name: vscode-git
+version: latest
+type: VS Code extension
+displayName: Git
+title: This extension is bundled with Visual Studio Code.
+description: Git SCM Integration.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/microsoft/vscode/tree/master/extensions/git
+category: Other
+firstPublicationDate: '2016-03-11'
```

Comment:
hello, AFAIK it's the date published on this registry no ?

Reply:
I thought that it is the first commit date of the related repository, will fix that to the date of merge.